knowledge/insects/worker_bees/gna.yml

```diff
@@ -0,0 +1,28 @@
+created_by: Kevin Kim
+domain: Nature
+seed_examples:
+- answer: |
+    Worker bee will progress through different roles during its lifcycle.  Worker bee will start as 
+    Cell Cleaner, Nursing Bee, Queen Attendant, Wax Production, and finally Foraging Bee.
+  question: What are the progressions in worker bee's lifecycle?
+- answer: |
+  In most common bee species, worker bees are infertile due to enforced altruistic kin selection,[23] and thus never reproduce. Workers are nevertheless considered female for anatomical and genetic reasons. Genetically, a worker bee does not differ from a queen bee and can even become a laying worker bee, but in most species will produce only male (drone) offspring. Whether a larva becomes a worker or a queen depends on the kind of food it is given after the first three days of its larval form.
+  question:  What are the genetic characteristics of worker bee?
+- answer: |
+    The worker bee's stinger is a complex organ that allows a bee to defend itself and the hive from most mammals.[26] Bee stings against mammals and birds typically leave the stinger embedded in the victim due to the structure of flesh and the stinger's barbs. In this case, the venom bulb stays with the stinger and continues to pump. The bee will die after losing its stinger, as the removal of the stinger and the venom bulb damages or removes other internal organs as well.
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!